packages/mui-material/src/Divider/Divider.js

```diff
@@ -68,7 +68,9 @@ const DividerRoot = styled('div', {
       width: '100%',
     }),
     ...(ownerState.light && {
-      borderColor: alpha(theme.palette.divider, 0.08),
+      borderColor: theme.vars
+        ? `rgba(${theme.vars.palette.divider} / 0.08)`
```

Comment:
```suggestion
        ? `rgba(${theme.vars.palette.dividerChannel} / 0.08)`
```
Let's wait the code from #32128

Reply:
Sorry for the delay, left my last comment on https://github.com/mui/material-ui/pull/32128 :)

Reply:
@vicasas Can you pull the latest master and add `dividerChannel` to `experimental_extendTheme.js`?

Reply:
@siriwatknp I just did it. I think I missed adding it to the types...

Could it be like this?

```diff
export interface PaletteWithChannels {
  ...
  divider: TypeDivider;
+ dividerChannel: TypeDivider;
  action: TypeAction & { disabledChannel: string };
  ...
}
```

Reply:
Agree!